.github/workflows/test-registration1-e2e.yaml

```diff
@@ -114,6 +156,25 @@ jobs:
         env:
           HAPPO_API_KEY: ${{ secrets.HAPPO_API_KEY }}
           HAPPO_API_SECRET: ${{ secrets.HAPPO_API_SECRET }}
-          HAPPO_NONCE: ${{ github.sha }}-reg1
+          HAPPO_NONCE: ${{github.run_id}}${{ github.run_attempt}}-reg1
         run: npx happo-e2e finalize
         working-directory: ./bciers/apps/registration1
+
+  # Call Happo api and skip the project if it wasn't affected
+  happo-skip-not-affected:
+    if: ${{ !inputs.is_nx_affected && github.ref_name != 'develop' && github.ref_name != 'main' && github.head_ref != 'develop' && github.head_ref != 'main' }}
```

Comment:
I put this back here even though we aren't skipping reg1 if not affected yet (still waiting on a team consensus 🤓 )